clients/drcachesim/docs/drcachesim.dox.in

```diff
@@ -1010,48 +1010,109 @@ architecture to achieve optimal efficiency.  Google is sharing
 instruction and memory address traces from workloads running in Google
 data centers so that computer architecture researchers can study and
 develop new architecture ideas to improve the performance and
-efficiency of this important class of workloads.
+efficiency of this important class of workloads. To protect Google's
+intellectual property, these traces have been modified in order to
+filter out sensitive information. These traces follow a synthetic ISA
```

Comment:
Do we want to separate Version 1 from Version 2 so someone who still has the v1 trace won't be confused?  Even leaving this all as is but adding a "Deprecated Version 1 Format" section at the bottom w/ a brief description and a link from the top of the file may be enough.